bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/BlindHandler.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dominoswiss.internal;
+
+import static org.openhab.binding.dominoswiss.internal.DominoswissBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link BlindHandler} is responsible for handling commands, which are
+ * sent to one of the channels.The class defines common constants, which are
+ * used across the whole binding
+ *
+ * @author Frieso Aeschbacher - Initial contribution
+ */
+@NonNullByDefault
+public class BlindHandler extends BaseThingHandler {
+
+    private Logger logger = LoggerFactory.getLogger(BlindHandler.class);
+
+    private @Nullable EGateHandler dominoswissHandler;
+
+    private String id = "";
+
+    public BlindHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Blind got command: {} and ChannelUID: {} ", command.toFullString(),
+                channelUID.getIdWithoutGroup());
+        dominoswissHandler = (EGateHandler) getBridge().getHandler();
```

Comment:
`getBridge()` can return null. You can store the return value to a local variable and check if it is null to get rid of the compiler warning.